### PR TITLE
Add system robot account UI

### LINKED ISSUE #13295: Robot account enhancement UI work
What can we help you?

### PR DESCRIPTION
**Pull request by AllForNothing**
Fixes #13295
UX:  https://www.figma.com/proto/cPY9Lh8LbIQawAJSGmi1KA/Harbor-v2.0?node-id=1989%3A0&viewport=6462%2C5249%2C1.0763888359069824&scaling=min-zoom

Temp Env: https://10.185.12.35
Changes:
image
image

Signed-off-by: AllForNothing <sshijun@vmware.com>

### CODE REVIEW

File: src/portal/src/app/system-robot-accounts/new-robot/new-robot.component.ts [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+    // tslint:disable-next-line:triple-equals
+    if (this.systemRobot.duration != this.originalRobotForEdit.duration) {
+      return true;
+    }
+    // tslint:disable-next-line:triple-equals
+    if (this.systemRobot.description != this.originalRobotForEdit.description) {
+      return true;
+    }
+    // tslint:disable-next-line:triple-equals
+    if (this.coverAll != this.coverAllForEdit) {
```
Comment (jwangyangls) [inline src/portal/src/app/system-robot-accounts/new-robot/new-robot.component.ts:284]: !==

File: src/portal/src/app/project/robot-account/add-robot/add-robot.component.scss [hunk 2]
```diff
@@ -52,4 +21,28 @@
...
+  color: green;
+}
+.dropdown-per {
+  margin-left: -12px;
+}
+.mt-description {
+  width: 238px;
+}
+.mt-10px {
+  margin-top: 8px !important;
```
[the remark this answers is not in the document]
Comment (AllForNothing) [reply src/portal/src/app/project/robot-account/add-robot/add-robot.component.scss:47]: modified

File: src/portal/src/app/system-robot-accounts/new-robot/new-robot.component.ts (hunk 1 above)
Comment (AllForNothing) [reply src/portal/src/app/system-robot-accounts/new-robot/new-robot.component.ts:284]: modified

**Review by jwangyangls — APPROVED** [review]
lgtm

**Review by xuelichao — APPROVED** [review]

**Pull request merged**